plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<?eclipse version="3.4"?>
+<?eclipse version="3.2"?>
```

Comment:
Um, I think the docs aren't up to date.  Creating a new plug-in with PDE has 3.4:

```
<?eclipse version="3.4"?>
```